<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-L2UM9ODI" failures="1" tests="5" name="LoginTest" time="41.458" errors="0" timestamp="2025-08-29T21:07:13 IST">
  <testcase classname="Test.LoginTest" name="makeAppointment" time="1.266"/>
  <testcase classname="Test.LoginTest" name="invalidLogin1" time="4.06">
    <failure type="java.lang.AssertionError" message="expected [Mahesh] but found [https://katalon-demo-cura.herokuapp.com/profile.php#login]">
      <![CDATA[java.lang.AssertionError: expected [Mahesh] but found [https://katalon-demo-cura.herokuapp.com/profile.php#login]
at Test.LoginTest.invalidLogin1(LoginTest.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin1 -->
  <testcase classname="Test.LoginTest" name="invalidLogin2" time="4.349"/>
  <testcase classname="Test.LoginTest" name="login" time="2.718"/>
  <testcase classname="Test.LoginTest" name="ValidForm" time="4.565"/>
</testsuite> <!-- LoginTest -->
